packages/blazor-workspace/NimbleBlazor/Components/NimbleWaferMap.razor.cs

```diff
@@ -135,6 +138,19 @@ public partial class NimbleWaferMap : ComponentBase
         }
     }
 
+    /// <summary>
+    /// Sets the data in the wafer map.
+    /// </summary>
+    /// <param name="data">The input data, an Apache.Arrow.RecordBatch, which fills the wafer map with content</param>
+    public async Task SetDataAsync(RecordBatch data)
```

Comment:
I think we need two overloads, one that takes the apache record format and one that takes the previous dies format.